# Cálculo - Tarefa 02

1- Calcule de forma numérica a integral da função $x^3$ avaliada entre os pontos $1$ e $2$. Identifique o número de retângulos que você usou.  
2- Calcule essa área de forma analítica.

In [1]:
import pandas as pd
import numpy as np

In [1]:
from scipy import integrate

# Defina a função a ser integrada
def f(x):
    return x**3

# Calcule a integral definida
result, error = integrate.quad(f, 1, 2)

# Imprima o resultado
print(f"O resultado da integral definida é: {result}")

O resultado da integral definida é: 3.7500000000000004



**Cálculo Numérico:**

Para calcular numericamente a integral da função x³ entre os pontos 1 e 2,usando a regra dos retangulos: 

 Primeiro, determinamos a largura dos retangulos(Δx):

 Δx = $\frac{ponto \ final - ponto \ inicial}{\\   numero \ do \ retangulo }$


No intevalo de 1 a 2,Δx =  $\frac{2-1 }{\\n }$,onde n é o numero de retangulos.

A area de cada retangulo é Δx X f (ponto medio), onde f(x) é a função x³

Por exemplo, paraa n = 4:

Δx = $\frac{2-1}{\\ 4  }$ = $\frac{1}{\\ 4 }$ = 0.25

Os ponos medios para n = 4 são 1.125,1.375, 1.625 e 1.875.

Área Total = (Δx) X [ f (1.125)+ f(1.375) + f(1.625)+ f (1.875)]

**Cálculo Analitico:**

Para calcular a integral de x³ de forma analítica entre 1 e 2:
$$\int_a^b x^n \,dx = \frac{{1}x⁴}{4} \Bigg|_1^2$$


Portanto, a integral de x3 entre 1 e 2 é  3.75.






